### PR TITLE
Wpilib article 2024 updates

### PR DESCRIPTION
**Pull request by Lewis-Seiden**

### CODE REVIEW

File: Docs/General/ElectronicsCrashCourse.md [hunk 1]
```diff
@@ -76,10 +80,21 @@ When you start up the robot you may see red error messages in the drive station
...
 
 We have to set the IDs of the devices on the CAN network to make sure they are all uniquely and correctly identified.
-It would be very awkward to try to drive the robot and accidentally run the elevator!
+It would be very awkward to try to drive the robot and accidentally run the elevator instead of the drivetrain!
 To set the IDs of devices on the CAN network we use the [Tuner X](https://pro.docs.ctr-electronics.com/en/stable/docs/tuner/index.html) app built by CTRE.
 This app also has features to test and configure devices on the CAN network.
 
+### CANivore
+
+![CANivore](../../Assets/CANivore.webp)
```
Comment (spellingcat) [inline Docs/General/ElectronicsCrashCourse.md:89]: this is not that important but could you make the images a bit smaller or at least around the same size

File: Docs/General/ElectronicsCrashCourse.md [hunk 2]
```diff
@@ -76,10 +80,21 @@ When you start up the robot you may see red error messages in the drive station
 If you see that, something might be broken on the CAN network.
```
Comment (spellingcat) [inline Docs/General/ElectronicsCrashCourse.md:80]: what is being communicated, what happens to the rest of the network if one goes down?

File: Docs/General/ElectronicsCrashCourse.md [hunk 3]
```diff
@@ -100,37 +115,54 @@ These ports include:
...
 
-![Radio](../../Assets/Radio.jpg)
+![Radio](../../Assets/Radio.png)
 
 The radio is essentially a Wi-Fi router that connects the robot to the driver station.
 At tournaments we have to take the radio to get reprogrammed for that competition.
-This makes it able to connect to the field so that our robot gets enabled and disabled at the right times during matches, however it prevents us from connecting to the robot with a laptop wirelessly.
-The radio has two ethernet ports and one barrel jack port.
-One ethernet port connects to the rio.
+This makes it able to connect to the field so that our robot gets enabled and disabled at the right times during matches, however it prevents us from connecting to the robot with a laptop wirelessly$^1$.
```
Comment (spellingcat) [inline Docs/General/ElectronicsCrashCourse.md:135]: idt the footnotes rendered correctly

File: Docs/General/ElectronicsCrashCourse.md [hunk 4]
```diff
@@ -197,11 +232,11 @@ The CANivore gets around some hardware limitations of the Rio to have extra band
 Encoders are devices that measure rotation.
 We use them to measure the angles of our swerve modules, the speed of our wheels, the position of our elevator, and much more.
 Modern motors have encoders built in.
-However these encoders are relative encoders.
-Relative encoders are set to 0 when the robot is turned on, and track the change in position from there.
-If all we care about measuring is velocity this is fine, but if we wanted to know how far our elevator is extended or what angle our swerve module is at we would need to align them to some known position whenever we started up our robot.
-Instead we can use absolute encoders to measure the absolute position of a mechanism.
```
Comment (spellingcat) [inline Docs/General/ElectronicsCrashCourse.md:203]: i feel like we should keep this

File: Docs/General/ElectronicsCrashCourse.md (hunk 1 above)
Comment (Lewis-Seiden) [reply Docs/General/ElectronicsCrashCourse.md:89]: idk if i can do formatting let me take a look

File: Docs/General/ElectronicsCrashCourse.md (hunk 4 above)
Comment (Lewis-Seiden) [reply Docs/General/ElectronicsCrashCourse.md:203]: the main thing is that they arent actually relative, and we likely wont encounter relative encoders anymore

**Review by spellingcat — APPROVED** [review]

**Pull request merged**